include/spm/fadedrv.h

```diff
@@ -1,10 +1,26 @@
 #pragma once
+#include <wii/types.h>
 
 namespace spm::fadedrv {
 
+struct FadeWork {
```

Comment:
Might be worth calling this FadeWork2

Reply:
Due to new discoveries this will remain FadeWork